amethyst_controls/src/systems.rs

```diff
@@ -229,7 +237,7 @@ impl<'a> System<'a> for CursorHideSystem {
             match *event {
                 Event::WindowEvent { ref event, .. } => match event {
                     &WindowEvent::Focused(focused) => {
-                        if focused {
+                        if focused && !self.released {
```

Comment:
What if the toggled value changes without a focus event? Does that mean the cursor will be released only on the next focus change?